etl-pipeline/.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: https://editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Maybe this should be Python-specific under etl-pipeline?
```

Comment:
hmm - since this is inside the etl-pipeline? Should it only look at files within this folder?

Reply:
d'oh yes, I meant to put this at the top level. Though I think the way this thing is read is relative to the dir it is in, so I think you don't need to specify `etl-pipeline`. I think I'll just leave this as is for now, and we can look into moving this up to the rootdir if someone wants to share whitespace rules with the web code.